Website/src/rendering/src/components/SessionsGrid.tsx

```diff
@@ -25,6 +44,41 @@ type SessionsGridProps = ComponentProps & {
   };
 };
 
+function getTimeString(time: number, isEndTime: boolean) {
+  const minutes = isEndTime ? ':55' : ':00';
+  time = isEndTime ? time - 1 : time;
+  if (time == 12) {
+    return time + minutes + ' noon';
+  } else if (time < 12) {
+    return time + minutes + ' am';
+  } else if (time > 12) {
+    return time - 12 + minutes + ' pm';
+  }
+  return '';
+}
+
+function GetSessionTime(timeslots: Timeslot[]) {
```

Comment:
I think this is repeated in another file. Please move to a utility file if that is the case.